moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/tesseract/basic_types.h

```diff
@@ -0,0 +1,106 @@
+/*********************************************************************
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2013, Southwest Research Institute
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *********************************************************************/
+
+/* Author: Levi Armstrong */
+
+#ifndef MOVEIT_COLLISION_DETECTION_BULLET_TESSERACT_BASIC_TYPES_H_
```

Comment:
There are still some files left with tesseract in their file path and header guard.
I'm fine with this, if you want to keep those as a reminder to migrate later.
Otherwise, these shoult be renamed as well, shouldn't they?

Reply:
Yes, we discussed the name `bullet_integration` as a subfolder for the old tesseract code. Any objections to the name?

Reply:
Changed it to `bullet_integration` including in all header guards